.gitignore

```diff
@@ -17,4 +17,5 @@ obs-package-from-git
 stamp-h1
 py-compile
 test-driver
+testsuite/*.txt.tmp
```

Comment:
NP: maybe a generic `testsuite/*.tmp` (or even global `*.tmp`) would be better (future proof)...